macro/Version.py

```diff
@@ -47,10 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'jade':
+        if version.lower() == 'kame':
```

Comment:
Let's use kinetic, though I really like kame more because it'll translate to turtle turtle in Japanese :-)